loadgen/contract/agent-trade-amm.js

```diff
@@ -13,12 +13,23 @@ export default async function startAgent([key, home]) {
 
   console.error(`trade-amm: building tools`);
   // const runIssuer = await E(agoricNames).lookup('issuer', issuerPetnames.RUN);
-  const { runBrand, bldBrand, amm, autoswap, runPurse, bldPurse } = await allValues({
+  const {
+    runBrand,
+    bldBrand,
+    amm,
+    autoswap,
+    runPurse,
+    bldPurse,
+  } = await allValues({
     runBrand: E(agoricNames).lookup('brand', issuerPetnames.RUN),
     // bldBrand: E(agoricNames).lookup('brand', issuerPetnames.BLD),
     bldBrand: E(E(wallet).getIssuer(issuerPetnames.BLD)).getBrand(),
-    amm: E(agoricNames).lookup('instance', 'amm'),
-    autoswap: E(agoricNames).lookup('instance', 'autoswap'),
+    amm: E(agoricNames)
+      .lookup('instance', 'amm')
+      .catch(() => {}),
+    autoswap: E(agoricNames)
+      .lookup('instance', 'autoswap')
+      .catch(() => {}),
```

Comment:
I thought of doing .lookup().catch(), but  guessed that Promise.any would succeed. Thanks for checking it out.  This looks good to me.